.github/workflows/ci.yml

```diff
@@ -119,6 +119,9 @@ jobs:
             - name: Check out app
               uses: actions/checkout@v3
 
+            - name: Install povray
+              run: sudo apt-get install povray
```

Comment:
Is the `povray` in the `full-stack`? I remember you proposed to add it?

Reply:
Yes, povray is in the image. But this is needed here for the unit tests, since they do not run in the container.